azure-pipelines.yml

```diff
@@ -1,21 +1,29 @@
 strategy:
   matrix:
+    Node 8 / Linux:
+      nodeVersion: '8.12.x'
+      imageName: 'ubuntu-16.04'
+      IPFS_URL: 'https://dist.ipfs.io/go-ipfs/v0.4.19/go-ipfs_v0.4.19_linux-amd64.tar.gz'
     Node 8 / macOS:
-      nodeVersion: '8.x'
+      nodeVersion: '8.12.x'
```

Comment:
Why `8.12.x` instead of `8.x`?  Does `8.x` pick up the latest minor in the version 8 series?